pkgs/applications/misc/postrunner/default.nix

```diff
@@ -0,0 +1,16 @@
+{ lib, bundlerApp }:
+
+bundlerApp {
+  pname = "postrunner";
```

Comment:
version is missing.

Reply:
When using bundlerApp (see also https://nixos.org/manual/nixpkgs/stable/#sec-language-ruby), I think that a version field is not needed, as bundlerApp gets the version from the gemset.nix.
I checked other ruby apps (gollum, jekyll) and both do not have a version in ther default.nix.
This makes updating the the version easier, too.

Reply:
you are right.